src/js/ar.js

```diff
@@ -138,7 +138,7 @@ const ar = {
         request: {
           configName: 'stockMovement',
           label: 'Request Stock',
-          link: '/openboxes/stockMovement/createRequest',
+          link: '/openboxes/stockMovement/createRequest?type=REQUEST',
```

Comment:
I don't like this. What's the point of this type param if you have createRequest...